src/chunk_dispatch_state.c

```diff
@@ -53,6 +53,8 @@ on_chunk_insert_state_changed(ChunkInsertState *cis, void *data)
 	ChunkDispatchState *state = data;
 	ModifyTableState *mtstate = state->mtstate;
 
+	if (mtstate == NULL)
```

Comment:
could you add a comment here that this may be NULL on pg12 under certain conditions

Reply:
Along the same lines, I would like to understand what is going on and why it is OK to just do nothing here before accepting this fix.